lib/src/migrators/module/unreferencable_members.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+// The sass package's API is not necessarily stable. It is being imported with
+// the Sass team's explicit knowledge and approval. See
+// https://github.com/sass/dart-sass/issues/236.
+import 'package:sass/src/ast/sass.dart';
+
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p;
+
+import 'package:sass_migrator/src/utils.dart';
+import 'unreferencable_type.dart';
+
+/// Tracks members that are unreferencable in the current scope.
+class UnreferencableMembers {
+  final UnreferencableMembers parent;
+  final _unreferencable = <SassNode, UnreferencableType>{};
+
+  UnreferencableMembers([this.parent]);
+
+  /// Marks [declaration] as unreferencable with the given [type].
+  markUnreferencable(SassNode declaration, UnreferencableType type) {
```

Comment:
Make these members explicitly `void`.

Also, the `Unreferencable` in these method names is redundant with the class name (and in practice, the instance variable name as well). I'd call this method just `add()`, to evoke a `Set` (which this is basically just a fancy version of), and I'd rename `checkUnreferencable` to just `check()`.